helm/charts/nats/templates/configmap.yaml

```
@@ -354,6 +354,10 @@ data:
       {{- else }}
       no_tls: {{ .Values.websocket.noTLS }}
       {{- end }}
+      same_origin: {{ .Values.websocket.sameOrigin }}
+      {{- with .Values.websocket.allowedOrigins }}
+      allowed_origins: ["{{ first . }}" {{- range (rest .) }}, "{{ . }}" {{- end }}]
```

Might be simpler to use `toRawJson` so that it becomes the array? 

```suggestion
      allowed_origins: {{ toRawJson . }}
```

---

> Might be simpler to use `toRawJson` so that it becomes the array?

@wallyqs, thanks for the suggestion. I've updated it. PTAL